react-sharing-state/my-app/src/Container.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+
+/**
+ * A container of items.
+ * One item is displayed at a time, with buttons to flip through them:
+ * - Next and Prev scroll forward and backward one item
+ * - An array of buttons scroll to a specific item
+ * TODO: The buttons don't work correctly!
+ */
+export default function Container({ items }) {
+  const [current, setCurrent] = useState(0);
+  const [backgroundColor, setBackgroundColor] = useState('white');
```

Comment:
You don't use this anywhere. My expectation was that this be removed from the code entirely, which you did! -- but you left this unused state in the code.